fecgov/openFEC
# Feature/switch schedule b

## Summary (required)
Issus #3045 switch to use fecp-driven sched_b tables

## How to test the changes locally
After the switch, the schedule_b queries will be pointed to fecp-driven fec_fitem_sched_b tables.  When run the application locally, try api for schedule_b 
http://localhost:5000/developers/#!/disbursements/get_schedules_schedule_b
Make sure the API is still working. 

## Impacted areas of the application
After the switch, the schedule_b queries will be pointed to fecp-driven fec_fitem_sched_b tables.  When run the application locally, try api for schedule_b.  The will impact schedule_b related API.  However, the changes should be transparent to the users.
-  



## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
